examples/grid_queries/src/main.rs

```diff
@@ -0,0 +1,68 @@
+use chrono::{Utc, DateTime};
+use rust_connector_api::APIClient;
+use rust_connector_api::location::BBox;
+use rust_connector_api::errors::ConnectorError;
+use polars::prelude::*;
+
+#[tokio::main]
+async fn main(){
+    // Credentials  
+    let u_name: String = String::from("NA");
+    let u_pw: String = String::from("NA");
+    // let u_name: String = String::from("python-community");
+    // let u_pw: String = String::from("Umivipawe179");
+
+    // Create Client
+    let api: APIClient = APIClient::new(u_name,u_pw,10);
+    let df_ts = example_request(&api).await.unwrap();
+
+    // Print the query result
+    println!("{:?}", df_ts);
+
+    // Do some calculations
+    for col in vec!["t_2m:C", "precip_1h:mm"] {
+        let mean: f64 = df_ts[col].mean().unwrap();
```

Comment:
I would now use unwraps in a production code. Maybe expect, or handle this to show why the error occurred in the 1st place.